add cloud release notes (#14870)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## September 19, 2023

**General changes**

> - Remove 2 vCPU TiDB and TiKV nodes from TiDB Dedicated clusters.

Reviewer: ```suggestion
- Remove 2 vCPU TiDB and TiKV nodes from [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters.
```

> - Remove 2 vCPU TiDB and TiKV nodes from TiDB Dedicated clusters.

Reviewer 2: ```suggestion
- Remove 2 vCPU TiDB and TiKV nodes from [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters.
```

    The 2 vCPU option is no longer available on the **Create Cluster** page or the **Modify Cluster** page.

> - Release [TiDB Cloud serverless driver (beta)](tidb-cloud/serverless-driver.md) for JavaScript.

Reviewer 2: ```suggestion
- Release [TiDB Cloud serverless driver (beta)](/tidb-cloud/serverless-driver.md) for JavaScript.
```

>     TiDB Cloud serverless driver for JavaScript allows you to connect to your TiDB Serverless cluster over HTTPS. It is particularly useful in edge environments where TCP connections are limited, such as [Vercel Edge Function](https://vercel.com/docs/functions/edge-functions) and [Cloudflare Workers](https://workers.cloudflare.com/).

Reviewer: ```suggestion
    TiDB Cloud serverless driver for JavaScript allows you to connect to your [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) cluster over HTTPS. It is particularly useful in edge environments where TCP connections are limited, such as [Vercel Edge Function](https://vercel.com/docs/functions/edge-functions) and [Cloudflare Workers](https://workers.cloudflare.com/).
```

    For more information, see [TiDB Cloud serverless driver (beta)](/tidb-cloud/serverless-driver.md).

**Console changes**

> - Users can get an estimation of cost in the **Usage This Month** panel or while setting up the spending limit.

Reviewer: ```suggestion
- For [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) clusters, you can get an estimation of cost in the **Usage This Month** panel or while setting up the spending limit.
```

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## September 19, 2023

**General changes**

- Remove 2 vCPU TiDB and TiKV nodes from TiDB Dedicated clusters.

    The 2 vCPU option is no longer available on the **Create Cluster** page or the **Modify Cluster** page.

> - Support using TiDB Serverless branching in Vercel preview deployments, with the TiDB Cloud Vercel integration.
> 
>     For details, refer to [Integrate TiDB Cloud with Vercel](/tidb-cloud/integrate-tidbcloud-with-vercel.md).
> 

Author: ```suggestion
```
